packages/manager/modules/pci/src/projects/project/serving/namespace/models/details/details.html

```diff
@@ -148,7 +152,7 @@
                                 >&nbsp;:
                             </label>
                             <oui-select
-                                class="oui-select_inline"
+                                class="oui-select_inline align-middle"
```

Comment:
**suggestion:** Instead of repositioning the `<oui-select>`, you can simply remove the extra `mt-3` helper on the sibling `<label>` element.

Please let me know if it suits your needs.

Reply:
I removed all `align-middle` and changed the `mt-3` to a `mt-1`. I still need it to look properly aligned.